handlers/meshery_application_handler.go

```diff
@@ -551,12 +551,17 @@ func githubRepoApplicationScan(
 	var mu sync.Mutex
 	ghWalker := walker.NewGit()
 	result := make([]models.MesheryApplication, 0)
+	fileName := path[strings.LastIndex(path, "/")+1:]
```

Comment:
From the implementation it looks like this function was made considering that `path` will be a directory i.e. when a user did `mesheryctl app onboard -f https://github.com/service-mesh-patterns/service-mesh-patterns/tree/master/samples/applications/emojivoto` it onboarded all the files in that directory but failed when a user provided a specific file `mesheryctl app onboard -f https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/samples/applications/emojivoto/k8s.yaml`. 

I've changed the implementation assuming that the path provided will be a file only. Thoughts? // @Revolyssup @alphaX86 @gr455